src/bp-templates/bp-legacy/buddypress/members/single/groups.php

```diff
@@ -11,9 +11,13 @@
 
 <div class="item-list-tabs no-ajax" id="subnav" aria-label="<?php esc_attr_e( 'Member secondary navigation', 'buddypress' ); ?>" role="navigation">
 	<ul>
-		<?php if ( bp_is_my_profile() ) bp_get_options_nav(); ?>
+		<?php
+		if ( bp_core_can_edit_settings() ) {
+			bp_get_options_nav();
+		}
+		?>
```

Comment:
Just use `bp_get_options_nav();` there, as explained in the corresponding comment about the Nouveau Template pack.

Reply:
Good point, thank you.